src/azure-cli-core/azure/cli/core/azclierror.py

```diff
@@ -13,6 +13,9 @@
 logger = get_logger(__name__)
 # pylint: disable=unnecessary-pass
 
+UNAUTHORIZED_MESSAGE = ("The access token has expired or been revoked due to being blocked by Continuous Access "
+                        "Evaluation. To re-authenticate, please run `az login`. "
```

Comment:
My understanding is continuous access evaluation just makes revoke happen before AT is expired. The statement can be "blocked by access policy" or "not meet the criteria to access this resource"